deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,36 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/* -- FLEXBOX on body - flex-direction - column -- */
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.flex-column-header, .flex-column-footer {
+  flex-basis: 100px;
+  flex-grow: 0;
+  flex-shrink: 0;
+}
+
+.flex-column-main {
+  flex-basis: auto;
+  flex-grow:1;
+}
+
+.flex-row-main {
+  display: flex;
+}
+
+.flex-row-article {
+  flex-grow: 1;
+}
+
+.flex-row-nav, .flex-row-aside {
+  flex-basis: 300px;
+  flex-shrink: 0;
+}
```

Comment:
ik heb hier flex-direction: row-reverse; gebruikt in plaats van order, misschien is het ook een idee. wat vindt jij hiervan?

Reply:
Ha Chaudhry,

Dat is inderdaad in dit geval de beste manier geweest. Ik wilde proberen hoe de property order werkt. Ik probeer Flexbox in mijn systeem te krijgen maar kan het daardoor soms ook te complex maken. Bedankt voor de tip